assets/tailwind.config.js

```diff
@@ -159,6 +159,11 @@ module.exports = {
     "border-beerus",
     "border-bulma",
     "border-trunks",
+    "border-spacing-y-0",
```

Comment:
Where this is used? Maybe we can have classes passed directly? In overall, I think we should not manually add new things here (and old things should be removed from this list)